### PR TITLE
Refactor student pages

### PR DESCRIPTION
**Pull request by T-S43**
I'm done making the student pages refactoring it would be nice to get some feedback.
Thanks in advance.

### CODE REVIEW

**Comment by T-S43** [comment]
Forgot to mention requesting an item and then ordering it doesn't work I didn't have enough time to finish it before the end of the sprint, due to scheduling I will add this to the backlog

File: app/views/student/bestellingen.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+}
+// choose between 1: super user,    2: warehouse-admin,     3: financial admin,     4: student
+
+$userId = $_SESSION['id'];
+// Left join for warehouse name or any other needed information
+$sql = "SELECT * FROM `request` LEFT JOIN `warehouse` ON (request.warehouseId = warehouse.id) WHERE $userId = `userId`";
+
+$result = mysqli_query($conn, $sql);
+// record is used for the table top row.
+$record = "";
```
Comment (QEUN) [inline app/views/student/bestellingen.php]: this line I would say should not be an echo and it should not be in the view to make it work becease u can make it come out with the rows u can make in the controller of this by using those tags there or if its fixed u can hold it above but using echo's seems kinda useles

Comment (T-S43) [reply app/views/student/bestellingen.php]: That is weird this is the old version of the bestellingen page and doesn't have echo's anymore

Comment (T-S43) [reply app/views/student/bestellingen.php]: I checked and it's not there anymore in the files

**Pull request merged**